File: JavaScript/1013.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');

const pegarValores = (line) => line.split(" ").map(a=>+a)
const[a,b,c] = pegarValores(lines.shift())

let maiorAB = (a+b+Math.abs(a-b))/2
let maiorFim = (maiorAB+c+Math.abs(maiorAB-c))/2
console.log(`${maiorFim} eh o maior`)

File: Python/1013.py
lista = (input().split(" "))
a,b,c = lista 
maior = (int(a) + int(b) + abs(int(a)-int(b)))/2
result = (int(maior) + int(c) + abs(int(maior) - int(c)))/2
print("%d eh o maior"%result)